site/docs/master/README.md

```diff
@@ -47,8 +47,8 @@ See [the list of releases][6] to find out about feature changes.
 [24]: https://groups.google.com/forum/#!forum/projectvelero
 [25]: https://kubernetes.slack.com/messages/velero
 
-[28]: install-overview.md
-[29]: https://velero.io/docs/master/
+[28]: advanced-installation.md
```

Comment:
I don't think [28] is used anywhere currently, did you intend to add a ref to it? If not let's delete it entirely

Reply:
I missed removing it from this file.